### PR TITLE
DEVOPS-218: make `scripts/aws.hs s3 set-daedalus-release-build` also provide version texts

### PR DESCRIPTION
**Pull request by deepfire**
Test scenario:

```
[nix-shell:~/iohk]$ scripts/aws.hs s3 --bucket daedalus-travis-test set-daedalus-release-build
 0.5.1994 0.5.2110.0 v9cnlytcf5my4g2u
Setting Daedalus release redirects to:
  OS X:   https://s3.eu-central-1.amazonaws.com/daedalus-travis/Daedalus-installer-0.5.1994.pkg
  Win64:  https://ci.appveyor.com/api/buildjobs/v9cnlytcf5my4g2u/artifacts/installers/daedalus-
win64-0.5.2110.0-installer.exe

============== Checking if OSX build URL is live:
Spider mode enabled. Check if remote file exists.
--2017-07-27 01:07:54--  https://s3.eu-central-1.amazonaws.com/daedalus-travis/Daedalus-install
er-0.5.1994.pkg
Resolving s3.eu-central-1.amazonaws.com (s3.eu-central-1.amazonaws.com)... 52.219.72.32
Connecting to s3.eu-central-1.amazonaws.com (s3.eu-central-1.amazonaws.com)|52.219.72.32|:443..
. connected.
HTTP request sent, awaiting response... 200 OK
Length: 84334287 (80M) [application/octet-stream]
Remote file exists.

Both URLs live, proceeding to update latest release build references to:
  OS X:  https://s3.eu-central-1.amazonaws.com/daedalus-travis/Daedalus-installer-0.5.1994.pkg
  Win64: https://ci.appveyor.com/api/buildjobs/v9cnlytcf5my4g2u/artifacts/installers/daedalus-win64-0.5.2110.0-installer.exe

{
    "ETag": "\"26ce8d1c8bf5bb7fd1a36ebc745a1872\""
}
Done.  Following URLs should be live within a minute:

  OS X:   http://daedalus-travis-test.s3-website.eu-central-1.amazonaws.com/daedalus-osx-latest.pkg
  Win64:  http://daedalus-travis-test.s3-website.eu-central-1.amazonaws.com/daedalus-win64-latest.exe
```

Version JSON (for `daedalus-travis-test` bucket): https://s3.eu-central-1.amazonaws.com/daedalus-travis-test/daedalus-latest-version.json

### CODE REVIEW

File: scripts/aws.hs [hunk 1]
```diff
@@ -27,25 +27,35 @@ ppURL (URL pro ho ke) = pro <> "://" <> ho <> "/" <> ke
 -- * Buckets
 data Bucket     = Bucket { fromBucket ∷ T.Text } deriving (Show)
 
-defaultBucket = Bucket "daedalus-travis"
+default'bucket = Bucket "daedalus-travis"
```
Comment (domenkozar) [inline scripts/aws.hs]: Please, let's not use this notation and use the old name

File: scripts/aws.hs [hunk 2]
```diff
@@ -105,9 +125,10 @@ runS3 CheckDaedalusReleaseURLs bucket = do
   printf ("  OS X:   "%s%"\n") (ppURL $ osxLive bucket)
   printf ("  Win64:  "%s%"\n") (ppURL $ w64Live bucket)
 
-runS3 (SetDaedalusReleaseBuild rsOSX@(OSX id) rsWin64@(Win64 avPath)) bucket = do
-  let (osxurl@(URL osxpr osxcn osxpath), w64url@(URL w64pr w64cn w64path)) =
-        (buildURL id, avpathURL avPath)
+runS3 (SetDaedalusReleaseBuild debug'mode rsOSX@(R_OSX osx'bid) rsWin64@(R_Win64 w64'bid avId)) bucket = do
+  let (   osxurl@(URL osxpr osxcn osxpath)
```
Comment (domenkozar) [inline scripts/aws.hs]: Can we name these variables to represent their values? It's really hard to follow the code with `osxpr` and `osxcn`, etc

File: scripts/stack.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+##
```
Comment (domenkozar) [inline scripts/stack.yaml]: Do we really need this given that we have top-level stack.yaml?

**Comment by domenkozar** [comment]
Needs a CI fix by running `pkgs/generate.sh`

File: scripts/stack.yaml (hunk 3 above)
Comment (deepfire) [inline scripts/stack.yaml]: This is here because of `intero`, yes -- there is no `stack.yaml` that covers `scripts/aws.hs`.
